### PR TITLE
Suppport new yq syntax

### PR DESCRIPTION
**Pull request by DebakelOrakel**

### CODE REVIEW

**Review by bastjan — COMMENTED** [review]
I guess we should add a minimum supported version to the prerequisites?

File: docs/modules/ROOT/pages/how-tos/recover-from-backup.adoc [hunk 1]
```diff
@@ -70,15 +70,15 @@ vault login -method=oidc
 +
 [source,console]
 ----
-export RESTIC_REPOSITORY=s3:$(yq read -cjd'*' catalog/manifests/cluster-backup/05_schedule.yaml | jq -r '.[] | select(.kind == "Schedule") | .spec.backend.s3 | .endpoint + "/" + .bucket')
+export RESTIC_REPOSITORY=s3:$(yq -o=json '.' catalog/manifests/cluster-backup/10_object.yaml | jq -r 'select(.kind == "Schedule")| .spec.backend.s3 | .endpoint + "/" + .bucket')
```
Comment (simu) [inline docs/modules/ROOT/pages/how-tos/recover-from-backup.adoc:73]: If we're depending on yq v4, we can drop the `jq` completely:

```suggestion
export RESTIC_REPOSITORY=s3:$(yq -o=json 'select(.kind == "Schedule")| .spec.backend.s3 | .endpoint + "/" + .bucket' catalog/manifests/cluster-backup/10_object.yaml)
```

**Review by corvus-ch — CHANGES_REQUESTED** [review]
Please update the prerequisites. Be specific on which version of `yq` to use.

As far as I am aware, there are different, incompatible version of `yq` out there. Could it be you use a different one, or did they change stuff?

**Comment by DebakelOrakel** [comment]
Added minimum version for `yq`

File: docs/modules/ROOT/pages/how-tos/recover-from-backup.adoc [hunk 2]
```diff
@@ -70,15 +70,15 @@ vault login -method=oidc
...
 
-PASSWORD_KEY="$(cat catalog/manifests/cluster-backup/* | yq read -cjd'*' - | jq -r '.[] | select(.kind == "Secret" and .metadata.name == "objects-backup-password") | .stringData.password' | cut -d: -f2)"
+PASSWORD_KEY="$(yq -o=json '.' catalog/manifests/cluster-backup/10_object.yaml | jq 'select(.kind == "Secret" and .metadata.name == "objects-backup-password") | .stringData.password' | cut -d: -f2)"
```
Comment (simu) [inline docs/modules/ROOT/pages/how-tos/recover-from-backup.adoc:75]:
```suggestion
PASSWORD_KEY="$(yq -o=json 'select(.kind == "Secret" and .metadata.name == "objects-backup-password") | .stringData.password' catalog/manifests/cluster-backup/10_object.yaml | cut -d: -f2)"
```

**Comment by DebakelOrakel** [comment]
> Please update the prerequisites. Be specific on which version of `yq` to use.
> 
> As far as I am aware, there are different, incompatible version of `yq` out there. Could it be you use a different one, or did they change stuff?

The syntax changed a while ago with v4

File: docs/modules/ROOT/pages/how-tos/recover-from-backup.adoc [hunk 3]
```diff
@@ -70,15 +70,15 @@ vault login -method=oidc
...
 export RESTIC_PASSWORD=$(vault kv get -format json "clusters/kv/${PASSWORD_KEY%/*}" | jq -r ".data.data.${PASSWORD_KEY##*/}")
 
-ID_KEY="$(cat catalog/manifests/cluster-backup/* | yq read -cjd'*' - | jq -r '.[] | select(.kind == "Secret" and .metadata.name == "object-backup-s3-credentials") | .stringData.username' | cut -d: -f2)"
+ID_KEY="$(yq -o=json '.' catalog/manifests/cluster-backup/10_object.yaml | jq 'select(.kind == "Secret" and .metadata.name == "objects-backup-s3-credentials") | .stringData.username' | cut -d: -f2)"
```
Comment (simu) [inline docs/modules/ROOT/pages/how-tos/recover-from-backup.adoc:78]:
```suggestion
ID_KEY="$(yq -o=json 'select(.kind == "Secret" and .metadata.name == "objects-backup-s3-credentials") | .stringData.username' catalog/manifests/cluster-backup/10_object.yaml | cut -d: -f2)"
```

File: docs/modules/ROOT/pages/how-tos/recover-from-backup.adoc [hunk 4]
```diff
@@ -70,15 +70,15 @@ vault login -method=oidc
...
 export AWS_ACCESS_KEY_ID=$(vault kv get -format json "clusters/kv/${ID_KEY%/*}" | jq -r ".data.data.${ID_KEY##*/}")
 
-SECRET_KEY="$(cat catalog/manifests/cluster-backup/* | yq read -cjd'*' - | jq -r '.[] | select(.kind == "Secret" and .metadata.name == "object-backup-s3-credentials") | .stringData.password' | cut -d: -f2)"
+SECRET_KEY="$(yq -o=json '.' catalog/manifests/cluster-backup/10_object.yaml | jq 'select(.kind == "Secret" and .metadata.name == "objects-backup-s3-credentials") | .stringData.password' | cut -d: -f2)"
```
Comment (simu) [inline docs/modules/ROOT/pages/how-tos/recover-from-backup.adoc:81]:
```suggestion
SECRET_KEY="$(yq -o=json 'select(.kind == "Secret" and .metadata.name == "objects-backup-s3-credentials") | .stringData.password' catalog/manifests/cluster-backup/10_object.yaml | cut -d: -f2)"
```

File: docs/modules/ROOT/pages/how-tos/recover-from-backup.adoc [hunk 5]
```diff
@@ -5,7 +5,7 @@
 * Executables used in this guide:
 ** `kubectl`
 ** `jq`
-** `yq`
+** `yq` (version 4 or higher)
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/how-tos/recover-from-backup.adoc:8]:
```suggestion
** https://mikefarah.gitbook.io/yq/v/v4.x/[`yq` (version 4 or higher)]
```

Please check if this works.

File: docs/modules/ROOT/pages/how-tos/recover-from-backup.adoc [hunk 6]
```diff
@@ -5,7 +5,7 @@
 * Executables used in this guide:
 ** `kubectl`
 ** `jq`
```
Comment (simu) [inline docs/modules/ROOT/pages/how-tos/recover-from-backup.adoc:7]:
```suggestion
```

**Review by corvus-ch — APPROVED** [review]

File: docs/modules/ROOT/pages/how-tos/recover-from-backup.adoc (hunk 5 above)
Comment (simu) [inline docs/modules/ROOT/pages/how-tos/recover-from-backup.adoc:8]:
```suggestion
** `yq` https://mikefarah.gitbook.io/yq[yq] (version 4 or higher)
```

**Review by simu — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/how-tos/recover-from-backup.adoc (hunk 1 above)
Comment (DebakelOrakel) [reply docs/modules/ROOT/pages/how-tos/recover-from-backup.adoc:73]: be my guest, hadn't had enough coffee for further refactoring 😅

**Review by simu — APPROVED** [review]

**Pull request merged**